desktop/test/render-visdiff.js

```diff
@@ -68,19 +77,29 @@ app.on('ready', () => {
   for (let i = 0; i < WORKER_COUNT; i++) {
     setTimeout(() => {
       console.log('Creating new worker window', i)
-      const workerWin = new BrowserWindow({show: false, width: CANVAS_SIZE, height: CANVAS_SIZE})
+      const workerWin = new BrowserWindow({show: DEBUG_WINDOWS, width: CANVAS_SIZE, height: CANVAS_SIZE, })
       console.log('Created new worker window', i)
 
       workerWin.on('ready-to-show', () => console.log('Worker window ready-to-show:', i))
-      workerWin.webContents.on('did-finish-load', () => console.log('Worker window did-finish-load:', i))
+      const firstDisplay = toRender.pop()
+      workerWin.webContents.on('did-finish-load', () => {
+        if (DEBUG_WINDOWS) {
+          workerWin.webContents.openDevTools('right')
+        }
+
+        console.log('Worker window did-finish-load:', i)
+        workerWin.webContents.send('load', {
```

Comment:
the shared html now assumes we get a load message with metadata so we inject the firstDisplay details in there. Hopefully this'll avoid some timing issues